health.tf

```diff
@@ -8,12 +8,16 @@
 #  worker_nodes         = local.worker_private_ipv4_list
 #}
 
+locals {
```

Comment:
Need to investigate: the value returned by terraform/hetzner is something like `2a01:4f8:c014:ec13::` but curl and other http libraries expect it to end with an `1` like in  `2a01:4f8:c014:ec13::1` and also to be enclosed in brackets, like in `https://[2a01:4f8:c014:ec13::1]:6443`.

Both `http.talos_health` and `kubeconfig_data` required this format to work properly.

I never read the ipv6 RFC to understand the difference between ending with `::` and `::1` but I would bet that is something related to it.